plasmapy/physics/transport.py

```diff
@@ -82,133 +107,1175 @@ def Coulomb_logarithm(T, n_e, particles, V=None):
     Debye length, electric fields from other particles will be
     screened out due to electrons rearranging themselves.
 
-    The choice of inner impact parameter is more controversial. There
-    are two main possibilities.  The first possibility is that the
-    inner impact parameter corresponds to a deflection angle of 90
-    degrees.  The second possibility is that the inner impact
-    parameter is a de Broglie wavelength, :math:`\lambda_B`
-    corresponding to the reduced mass of the two particles and the
-    relative velocity between collisions.  This function uses the
-    standard practice of choosing the inner impact parameter to be the
-    maximum of these two possibilities.  Some inconsistencies exist in
-    the literature on how to define the inner impact parameter [2]_.
-
-    Errors associated with the Coulomb logarithm are of order its
+    The choice of inner impact parameter is the distance of closest
+    approach for a 90 degree Coulomb collision [2]_, [3]_.
+
+    Errors associated with the classical Coulomb logarithm are of order its
     inverse. If the Coulomb logarithm is of order unity, then the
     assumptions made in the standard analysis of Coulomb collisions
     are invalid.
 
+    For dense plasmas where the classical Coulomb logarithm breaks
+    down there are various extended methods. These can be found
+    in D.O. Gericke et al's paper, which has a table summarizing
+    the methods [4]_. The GMS-1 through GMS-6 methods correspond
+    to the methods found it that table.
+
+    It should be noted that GMS-4 thru GMS-6 modify the Coulomb
+    logarithm to the form:
+
+    .. math::
+        \ln{\Lambda} \equiv 0.5 \ln\left(1 + \frac{b_{max}^2}{b_{min}^2} \right)
+
+    This means the Coulomb logarithm will not breakdown for Lambda < 0,
+    which occurs for dense, cold plasmas.
+
     Examples
     --------
     >>> from astropy import units as u
     >>> n = 1e19*u.m**-3
     >>> T = 1e6*u.K
     >>> particles = ('e', 'p')
     >>> Coulomb_logarithm(T, n, particles)
-    14.748259780491056
-    >>> Coulomb_logarithm(T, n, particles, 1e6*u.m/u.s)
+    14.545527226436974
```

Comment:
The differences between the old code, the current code, and the table of values listed in Chen are due to differences in how thermal velocity is defined.
Old code: `sqrt(3 k T / m)`
Current code: `sqrt(2 k T / m)`
Chen: `sqrt(k T / m)`

I checked this by quickly replacing thermal velocity in `_boilerPlate` with a 3 instead of 2 for the coefficient. And I get:

```
Expected: (new code)
    14.545527226436974
Got: (old code)
    14.748259780491056
```

The way I work around this when testing the values from Chen in `test_transport.py` is by adding a factor of `np.log(2)` to the Coulomb logarithm values he lists.

I think the best way to deal with this is just to mention these difference in thermal velocity definitions explicitly in the docstrings.